src/main/java/com/wcc/platform/domain/cms/pages/events/EventsPageMetaData.java

```diff
@@ -0,0 +1,13 @@
+package com.wcc.platform.domain.cms.pages.events;
+
+import com.wcc.platform.domain.cms.attributes.Contact;
+import com.wcc.platform.domain.cms.attributes.HeroSection;
+import com.wcc.platform.domain.cms.pages.Pagination;
+
+/**
+ * Events page metaData details.
+ *
+ * @param hero
+ * @param contact
+ */
+public record EventsPageMetaData(Pagination pagination, HeroSection hero, Contact contact) {}
```

Comment:
Why hero and contact are part of eventPageMetadata? would make more sense to be in EventPage?

Reply:
The metadata part of the page is - that do not change, while the data part is events that will be changing we do not want the entire page to refresh just the events section so that's my thought of putting hero section in the metadata. Do you agree ?

Reply:
Yes, you have a good point. The best way to solve this is to have 2 api's one for the page and another just for events. What do you think? 

Because Hero and Contact are not metadata. PaginationMetadata or PageMetadata have different meaning for me. 

I think if we break down in two API's one could be `pageEvents` and the other `cms/events` what do you think?